Release 0.5.1 branch stopped to build in CI

Bug Report
==========
<!--
Please fill in a a meaningful and concise title in the field above.
It helps us to classify the issue even  before reading the body and handling the
issue a bit better.

Examples:
  * Great: "Unable to select fleet in galaxy map"
  * Good: "Unable to click fleet"
  * Not really okay: "Issue with fleets"
  * Horrible: "HELP!!!"
-->


Environment
-----------
<!--
You don't need to provide the information in section when they are not useful in
the context of this issue.  For example a crashing hosted server may not need
information about the graphic card, but maybe depend on the operating system you
use.  When in doubt provide all information you have available.

You can find the FreeOrion version number in the lower right corner of the game
main screen.

When stating the operating system also note the version of the operating system
so, `Windows 8 Pro` instead of `Windows` or `Mac OSX Mountain Lion` instead of
`OSX`.
-->

* **FreeOrion Version**: `release-v0.5.1`
* **Operating System**: Windows
* **Fetched as** <!-- delete lines below that don't apply -->
  * Compiled from source


Description
-----------
<!--
Add a meaningful description of the bug you encountered.

If a screenshot or image helps to describe the issue content feel free to create
one.

Also attach the log files the game creates.  The log files The log files can be
found on
* Windows: %APPDATA%\FreeOrion
* MacOSX: ~/Library/"Application Support"/FreeOrion
* Linux: ${XDG_DATA_HOME:-~/.local/share}/freeorion
-->

Starting MSVC 19.43.34808.0 branch `release-v0.5.1` fails to build:

```
2025-03-03T17:08:11.1311333Z D:\a\freeorion\freeorion\universe\ValueRefs.cpp(75,45): error C3083: 'system_clock': the symbol to the left of a '::' must be a type [D:\a\freeorion\freeorion\build\freeorioncommon.vcxproj]
2025-03-03T17:08:11.1548847Z D:\a\freeorion\freeorion\universe\ValueRefs.cpp(75,59): error C2039: 'now': is not a member of 'std::chrono' [D:\a\freeorion\freeorion\build\freeorioncommon.vcxproj]
2025-03-03T17:08:11.1550463Z       C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.43.34808\include\__msvc_chrono.hpp(286,11):
2025-03-03T17:08:11.1551450Z       see declaration of 'std::chrono'
2025-03-03T17:08:11.1551947Z   
2025-03-03T17:08:11.1799372Z D:\a\freeorion\freeorion\universe\ValueRefs.cpp(75,59): error C3861: 'now': identifier not found [D:\a\freeorion\freeorion\build\freeorioncommon.vcxproj]
2025-03-03T17:08:11.2007711Z D:\a\freeorion\freeorion\universe\ValueRefs.cpp(76,44): error C2672: 'std::chrono::duration_cast': no matching overloaded function found [D:\a\freeorion\freeorion\build\freeorioncommon.vcxproj]
2025-03-03T17:08:11.2010022Z       C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.43.34808\include\__msvc_chrono.hpp(87,19):
2025-03-03T17:08:11.2011199Z       could be '_To std::chrono::duration_cast(const std::chrono::duration<_Rep,_Period> &) noexcept(<expr>)'
2025-03-03T17:08:11.2011673Z   
2025-03-03T17:08:11.2012286Z D:\a\freeorion\freeorion\universe\ValueRefs.cpp(76,20): error C2737: 'now_mins': const object must be initialized [D:\a\freeorion\freeorion\build\freeorioncommon.vcxproj]
2025-03-03T17:08:11.2013460Z D:\a\freeorion\freeorion\universe\ValueRefs.cpp(77,50): error C2672: 'std::chrono::duration_cast': no matching overloaded function found [D:\a\freeorion\freeorion\build\freeorioncommon.vcxproj]
2025-03-03T17:08:11.2014457Z       C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.43.34808\include\__msvc_chrono.hpp(87,19):
2025-03-03T17:08:11.2015179Z       could be '_To std::chrono::duration_cast(const std::chrono::duration<_Rep,_Period> &) noexcept(<expr>)'
2025-03-03T17:08:11.2015649Z   
2025-03-03T17:08:11.2643221Z D:\a\freeorion\freeorion\universe\ValueRefs.cpp(78,13): error C3536: 'now_mins': cannot be used before it is initialized [D:\a\freeorion\freeorion\build\freeorioncommon.vcxproj]
2025-03-03T17:08:11.2644604Z D:\a\freeorion\freeorion\universe\ValueRefs.cpp(78,24): error C3536: 'previous_mins': cannot be used before it is initialized [D:\a\freeorion\freeorion\build\freeorioncommon.vcxproj]
```

Compare two logs: 

Successful with MSVC 19.42.34436.0  https://github.com/freeorion/freeorion/actions/runs/13473791691/job/37650451873
Unsuccessful with MSVC 19.43.34808.0 https://github.com/freeorion/freeorion/actions/runs/13635539845/job/38113275016

Expected Result
---------------
Correct compilation


Steps to reproduce
------------------
<!--
If the error only manifests itself after doing a certain number of actions
please add a list of steps to reproduce the bug.  If the error only occurs in
a certain game state please be sure to add a save game to the issue.

To upload save games please pack them into a Zip archive (*.zip) first.
-->
* Start build on CI

--- a/universe/ValueRefs.cpp
+++ b/universe/ValueRefs.cpp
@@ -4,6 +4,7 @@
 #if __has_include(<charconv>)
   #include <charconv>
 #endif
+#include <chrono>
 #include <functional>
 #include <iomanip>
 #include <iterator>

